### PR TITLE
12.2.0 Changelog

### PR DESCRIPTION
**Pull request by mike-plummer**

### CODE REVIEW

File: content/_changelogs/12.2.0.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+_Released 12/20/2022_
+
+**Features:**
+
+- Added the ability to match on `resourceType` with
+  [`cy.intercept()`](https://docs.cypress.io/api/commands/intercept), and to see
+  the resource type of an intercepted request as `req.resourceType`. Addresses
+  [#14525](https://github.com/cypress-io/cypress/issues/14525).
+- Users working in React Component Testing projects can now generate a basic
+  spec file by selecting from the components that exist in their project.
```
Comment (emilyrohrbough) [inline content/_changelogs/12.2.0.md:12]:
```suggestion
  spec file  from the components that exist in their project.
```

File: content/_changelogs/12.2.0.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  appropriately causing all frames to be captured which slowed down tests. Fixes
+  [#23830](https://github.com/cypress-io/cypress/issues/23830).
+- Fixed an issue where the `query` object was not available on requests from
+  [`cy.intercept()`](https://docs.cypress.io/api/commands/intercept) once they
+  were yielded. Fixes
+  [#25088](https://github.com/cypress-io/cypress/issues/25088).
+- Fixed a regression introduced in
+  [Cypress 12](https://docs.cypress.io/guides/references/changelog#12-0-0) where
+  [`cy.get()`](https://docs.cypress.io/api/commands/get) would ignore a `null`
+  value for the 'withinSubject' option. Fixes
```
Comment (emilyrohrbough) [inline content/_changelogs/12.2.0.md:29]:
```suggestion
  value for the `withinSubject` option. Fixes
```

File: content/_changelogs/12.2.0.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  migration wizard. Fixes
+  [#25138](https://github.com/cypress-io/cypress/issues/25138).
+- Fixed an issue where the Cypress migration wizard would fail to run in
+  [global mode](https://docs.cypress.io/guides/guides/command-line#cypress-open-global)
+  on newer versions of Cypress. Addressed in
+  [#25138](https://github.com/cypress-io/cypress/issues/25138).
+- Fixed an issue with Angular Component Testing where a custom `srcRoot`
+  configuration would not be respected. Fixes
+  [#24827](https://github.com/cypress-io/cypress/issues/24827).
+- Fixed an issue with Angular Component Testing where URLs within SASS/SCSS
```
Comment (emilyrohrbough) [inline content/_changelogs/12.2.0.md:45]:
```suggestion
- Fixed an issue with Angular Component Testing where urls within SASS/SCSS
```

File: content/_changelogs/12.2.0.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+- Fixed an issue where the `query` object was not available on requests from
+  [`cy.intercept()`](https://docs.cypress.io/api/commands/intercept) once they
+  were yielded. Fixes
+  [#25088](https://github.com/cypress-io/cypress/issues/25088).
+- Fixed a regression introduced in
+  [Cypress 12](https://docs.cypress.io/guides/references/changelog#12-0-0) where
+  [`cy.get()`](https://docs.cypress.io/api/commands/get) would ignore a `null`
+  value for the 'withinSubject' option. Fixes
+  [#25104](https://github.com/cypress-io/cypress/issues/25104).
+- Fixed Typescript typings for
```
Comment (emilyrohrbough) [inline content/_changelogs/12.2.0.md:31]: Lets list this last-- its not as impactful as the of the other bug fixes in this list.

File: content/_changelogs/12.2.0.md (hunk 3 above)
Comment (emilyrohrbough) [reply content/_changelogs/12.2.0.md:45]: I'd also prob list this as number 2 or 3 in the list of fixes.

**Comment by emilyrohrbough** [comment]
FYI - https://github.com/cypress-io/cypress/pull/25059#pullrequestreview-1223227898 just merged

File: content/_changelogs/12.2.0.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+- Fixed a regression introduced in the Electron browser in
+  [Cypress 10.8.0](https://docs.cypress.io/guides/references/changelog#10-8-0)
+  where the `CYPRESS_EVERY_NTH_FRAME` environment variable was not being set
+  appropriately causing all frames to be captured which slowed down tests. Fixes
+  [#23830](https://github.com/cypress-io/cypress/issues/23830).
+
+**Bugfixes:**
+
+- Fixed an issue where the `query` object was not available on requests from
+  [`cy.intercept()`](https://docs.cypress.io/api/commands/intercept) once they
```
Comment (emilyrohrbough) [inline content/_changelogs/12.2.0.md:26]:
```suggestion
  [`cy.intercept()`](/api/commands/intercept) once they
```

File: content/_changelogs/12.2.0.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+  were yielded. Fixes
+  [#25088](https://github.com/cypress-io/cypress/issues/25088).
+- Fixed an issue with Angular Component Testing where urls within SASS/SCSS
+  files were not being correctly resolved which could result in incomplete
+  styling. Fixes [#24272](https://github.com/cypress-io/cypress/issues/24272).
+- Fixed a regression introduced in
+  [Cypress 12](https://docs.cypress.io/guides/references/changelog#12-0-0) where
+  [`cy.get()`](https://docs.cypress.io/api/commands/get) would ignore a `null`
```
Comment (emilyrohrbough) [inline content/_changelogs/12.2.0.md:34]: links to cypress-docs content shouldn't include `https://docs.cypress.io` 
```suggestion
  [`cy.get()`](/api/commands/get) would ignore a `null`
```

File: content/_changelogs/12.2.0.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+  value for the `withinSubject` option. Fixes
+  [#25104](https://github.com/cypress-io/cypress/issues/25104).
+- Fixed an issue where the incorrect Cypress version could be shown in the
+  migration wizard. Fixes
+  [#25138](https://github.com/cypress-io/cypress/issues/25138).
+- Fixed an issue where the Cypress migration wizard would fail to run in
+  [global mode](https://docs.cypress.io/guides/guides/command-line#cypress-open-global)
```
Comment (emilyrohrbough) [inline content/_changelogs/12.2.0.md:41]:
```suggestion
  [global mode](/guides/guides/command-line#cypress-open-global)
```

File: content/_changelogs/12.2.0.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+  on newer versions of Cypress. Addressed in
+  [#25138](https://github.com/cypress-io/cypress/issues/25138).
+- Fixed an issue with Angular Component Testing where a custom `srcRoot`
```
Comment (emilyrohrbough) [inline content/_changelogs/12.2.0.md:44]: Is there a link to `srcRoot`?

File: content/_changelogs/12.2.0.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+  configuration would not be respected. Fixes
+  [#24827](https://github.com/cypress-io/cypress/issues/24827).
+- Fixed Typescript typings for
+  [`cy.nextUntil()`](https://docs.cypress.io/api/commands/nextuntil) to include
```
Comment (emilyrohrbough) [inline content/_changelogs/12.2.0.md:48]:
```suggestion
  [`cy.nextUntil()`](/api/commands/nextuntil) to include
```

File: content/_changelogs/12.2.0.md [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+
+- Fixed an issue where the `query` object was not available on requests from
+  [`cy.intercept()`](https://docs.cypress.io/api/commands/intercept) once they
+  were yielded. Fixes
+  [#25088](https://github.com/cypress-io/cypress/issues/25088).
+- Fixed an issue with Angular Component Testing where urls within SASS/SCSS
+  files were not being correctly resolved which could result in incomplete
+  styling. Fixes [#24272](https://github.com/cypress-io/cypress/issues/24272).
+- Fixed a regression introduced in
+  [Cypress 12](https://docs.cypress.io/guides/references/changelog#12-0-0) where
```
Comment (emilyrohrbough) [inline content/_changelogs/12.2.0.md:33]:
```suggestion
  [Cypress 12](#12-0-0) where
```

File: content/_changelogs/12.2.0.md [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+  the resource type of an intercepted request as `req.resourceType`. Addresses
+  [#14525](https://github.com/cypress-io/cypress/issues/14525).
+- Users working in React Component Testing projects can now generate a basic
+  spec file from the components that exist in their project. Addresses
+  [#24008](https://github.com/cypress-io/cypress/issues/24008).
+
+**Performance:**
+
+- Fixed a regression introduced in the Electron browser in
+  [Cypress 10.8.0](https://docs.cypress.io/guides/references/changelog#10-8-0)
```
Comment (emilyrohrbough) [inline content/_changelogs/12.2.0.md:18]:
```suggestion
  [10.8.0](#10-8-0)
```

File: content/_changelogs/12.2.0.md [hunk 12]
```diff
@@ -0,0 +1,50 @@
+## 12.2.0
+
+_Released 12/20/2022_
+
+**Features:**
+
+- Added the ability to match on `resourceType` with
+  [`cy.intercept()`](https://docs.cypress.io/api/commands/intercept), and to see
```
Comment (emilyrohrbough) [inline content/_changelogs/12.2.0.md:8]:
```suggestion
  [`cy.intercept()`](/api/commands/intercept), and to see
```

**Review by BlueWinds — APPROVED** [review]

**Review by flotwig — APPROVED** [review]

**Pull request merged**